packages/big-design/src/components/Table/HeaderCell/HeaderCell.tsx

```diff
@@ -53,6 +56,30 @@ const InternalHeaderCell = <T extends TableItem>({
     );
   };
 
+  const renderTooltip = (columnLabel: string) => {
+    if (typeof tooltip === 'string' && tooltip.length > 0) {
+      return (
+        <Tooltip
+          id={tooltipId}
+          trigger={
+            <Box as="span" marginLeft="xxSmall">
+              <BaselineHelpIcon
+                aria-describedby={tooltipId}
+                size="medium"
+                title={`Tooltip icon for the ${columnLabel}`}
```

Comment:
```suggestion
  const renderTooltip = () => {
    if (typeof tooltip === 'string' && tooltip.length > 0) {
      return (
        <Tooltip
          id={tooltipId}
          trigger={
            <Box as="span" marginLeft="xxSmall">
              <BaselineHelpIcon
                aria-describedby={tooltipId}
                size="medium"
                title="Hover or focus for additional context."
```

Chatted with @jordan-massingill about this, and this gives the most context for someone using a screen reader. Ideally, this would be passed in via the consumer.

Reply:
@chanceaclark I have updated the title. Check it please.